packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -537,6 +549,27 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
         );
     }
 
+    private renderTransparencyToggle(): React.ReactNode {
+        const id: string = uniqueId("transparency");
+        return (
+            <div
+                className={get(this.props, "managedClasses.explorer_viewerControlRegion")}
```

Comment:
can we add managedClasses as a default empty object so that we don't need to use "get"?

Reply:
We will fix this in a separate PR

Reply:
Looks like a larger refactor would have to happen, disregard this comment.